### ISSUE TITLE
backspace can't delete when editing php files

### ISSUE THREAD

**Issue by ghost** [spf13/spf13-vim#380]
When I edit something.php and try to delete something using backspace,it give me the error like this:
"
zsh:1: command not found: phpm
Shell  has retuen 127
please return ..."

But I works fine when editing any other file.
I am using xterm but I could't reproduce this in Konsole.So I am not sure about what is the problem.

**Comment by spf13** [comment]
I have to look up why backspace is mapped to that function, but it's trying
to look up the current word in the php manual.

https://github.com/jimktrains/phpm

What modes does this happen in?

## 

Steve Francia
http://stevefrancia.com
http://spf13.com
http://twitter.com/spf13

**Comment by ghost** [comment]
It happens in insert mode,and this is my xterm config,http://paste.kde.org/776564/  but I don't 
think it would be releated just in case I post it here.

I have to look up why backspace is mapped to that function, but it's trying to look up the 
current word in the php manual. 

https://github.com/jimktrains/phpm 

What modes does this happen in? 

-- Steve Francia http://stevefrancia.com http://spf13.com http://twitter.com/spf13 

On Mon, Jun 17, 2013 at 3:31 AM, wych42 notifications@github.com wrote: 

> When I edit something.php and try to delete something using backspace,it > give me the 
> error like this: > " > zsh:1: command not found: phpm > Shell has retuen 127 > please return ..." 
> 
> > But I works fine when editing any other file. > I am using xterm but I could't reproduce this 
> > in Konsole.So I am not sure > about what is the problem. > > — > Reply to this email directly or 
> > view it on GitHubhttps://github.com/spf13/spf13-vim/issues/380 > . > 
> > —Reply to this email directly or view it on GitHub[1].[2]

---

[1] https://github.com/spf13/spf13-vim/issues/380#issuecomment-19573373
[2] 
https://github.com/notifications/beacon/NCL59_9GA3QF2L4R96N6UxYQxpYI37VGcd6K7IqD1P
EjYk9Mmv-mXWp25Y8MU7pf.gif

**Comment by lovelock** [comment]
Hiya, have you find the solution? I got this error when I ssh to my server, yet it works fine in my own archlinux box.
In PIV, <C-H> is binded to find the manual of the function under cursor ONLY in insert mode, but unfortunately in insert mode <C-H> equals Backspace, I think this is the question.

**Comment by nivren** [comment]
I got the same error. I guess you are using Xshell as the terminal tools. In this case, you should change the keyboard setting of Xshell.  Alt + p  ->  Terminal -> Keyboard -> Backspace Key Sequence, change it to "ASCII 127".

**Comment by lovelock** [comment]
Not really, it's because the PIV plugin, it maps ctrl h to search phpm for function name under the cursor, remove the line and everything will be ok
发自我的 iPhone

**Comment by nivren** [comment]
thanks. So, where can I find this key mapping?

**Comment by lovelock** [comment]
PIV/ftplugin/php.vim 
Comment line 109 out.

发自我的 iPhone

**Comment by rku4er** [comment]
I had similar problem with .php files in my Xterm. To resolve it please push "Ctrl+LeftMouseButton" in xterm window and then uncheck "Backarrow Key (BS/DEL)". 
To make this change permanent just add XTerm.VT100.backarrowKey: false to your ~/.Xdefaults.
Hope this will help!
